resource/keywords_bdd.robot

```diff
@@ -77,23 +77,15 @@ I proceed to checkout
 I should see the shopping cart page with the products and its data and values
     selenium.Wait Until Page Contains Element                                        xpath=//*[@id="center_column"]                                    timeout=8s
     selenium.Element Should Contain                                                  css:td.cart_description > p:nth-child(1) > a:nth-child(1)         Faded Short Sleeve T-shirts
-    #obter a posição horizontal do botão "Proceed to checkout"
-    ${total_position_y}                                                              selenium.Get Horizontal Position                                  xpath=//*[@id="center_column"]/p[2]/a[1]
-    # subtrai 550 da posição obtida
-    ${total_position_y}                                                              Evaluate                                                          ${total_position_y} - 550
-    #executa o scroll de 0 até a posição - 550
-    selenium.Execute Javascript                                                      window.scrollTo(0,${total_position_y})
+    Set Test Variable                                                                ${locator}                                                        xpath=//*[@id="center_column"]/p[2]/a[1]
+    Scroll to specifc Position                                                       ${locator}
```

Comment:
@alinebiath I suggest undo this change and add a comment explaining the motivation for that workaround.

Reply:
Done.